### ISSUE TITLE
ImportError: No module named social_django.models

### ISSUE THREAD

**Issue by palzuncoff** [st4lk/django-rest-social-auth#35]
When i tried makemigrations i take following error

```
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/django/core/management/__init__.py", line 327, in execute
    django.setup()
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/giverroot/django/gve1/lib/python2.7/site-packages/social/apps/django_app/default/models.py", line 1, in <module>
    from social_django.models import AbstractUserSocialAuth, UserSocialAuth, Nonce, Association, Code, DjangoStorage
ImportError: No module named social_django.models
```

**Comment by st4lk** [comment]
Please provide output of `pip freeze` command

**Comment by palzuncoff** [comment]
```
amqp==2.1.4
backports.shutil-get-terminal-size==1.0.0
billiard==3.5.0.2
celery==4.0.1
decorator==4.0.10
Django==1.10.4
django-braces==1.10.0
django-filter==1.0.1
django-oauth-toolkit==0.11.0
django-rest-framework-social-oauth2==1.0.4
django-social-auth==0.7.28
djangorestframework==3.5.3
enum34==1.1.6
httplib2==0.9.2
ipython==5.1.0
ipython-genutils==0.1.0
kombu==4.0.1
Markdown==2.6.7
MySQL-python==1.2.5
oauth2==1.9.0.post1
oauthlib==1.1.2
pathlib2==2.1.0
pexpect==4.2.1
pickleshare==0.7.4
Pillow==3.4.2
prompt-toolkit==1.0.9
ptyprocess==0.5.1
Pygments==2.1.3
PyJWT==1.4.2
python-openid==2.2.5
python-social-auth==0.3.1
pytz==2016.10
requests==2.12.4
requests-oauthlib==0.7.0
rest-social-auth==0.5.0
simplegeneric==0.8.1
six==1.10.0
social-auth-core==0.0.1
traitlets==4.3.1
vine==1.1.3
wcwidth==0.1.7

```

**Comment by st4lk** [comment]
Fixed in #37
